### ISSUE TITLE
Model conversion from PyTorch to TF2 doesn't work properly for ALBERT

### ISSUE THREAD

**Issue by singletongue** [huggingface/transformers#3386]
# 🐛 Bug

## Information

The model conversion script `convert_pytorch_checkpoint_to_tf2.py` seems to be not working properly for ALBERT models.
It fails on the pre-trained models officially released by Google which are converted to PyTorch models with `convert_albert_original_tf_checkpoint_to_pytorch.py`.

## To reproduce

```
$ wget https://storage.googleapis.com/albert_models/albert_base_v2.tar.gz
$ tar xzf albert_base_v2.tar.gz
$ cd albert_base/
$ python -m transformers.convert_albert_original_tf_checkpoint_to_pytorch --tf_checkpoint_path model.ckpt-best --pytorch_dump_path ./pytorch_model.bin --albert_config_file albert_config.json
$ python -m transformers.convert_pytorch_checkpoint_to_tf2 --tf_dump_path ./ --model_type albert --pytorch_checkpoint_path ./pytorch_model.bin --config_file albert_config.json --compare_with_pt_model
...
Max absolute difference between models outputs 17.709423065185547
Traceback (most recent call last):
  File "/home/m-suzuki/.pyenv/versions/3.7.4/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/m-suzuki/.pyenv/versions/3.7.4/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/m-suzuki/.pyenv/versions/Python-3.7.4/lib/python3.7/site-packages/transformers/convert_pytorch_checkpoint_to_tf2.py", line 499, in <module>
    only_convert_finetuned_models=args.only_convert_finetuned_models,
  File "/home/m-suzuki/.pyenv/versions/Python-3.7.4/lib/python3.7/site-packages/transformers/convert_pytorch_checkpoint_to_tf2.py", line 428, in convert_all_pt_checkpoints_to_tf
    compare_with_pt_model=compare_with_pt_model,
  File "/home/m-suzuki/.pyenv/versions/Python-3.7.4/lib/python3.7/site-packages/transformers/convert_pytorch_checkpoint_to_tf2.py", line 351, in convert_pt_checkpoint_to_tf
    assert diff <= 2e-2, "Error, model absolute difference is >2e-2: {}".format(diff)
AssertionError: Error, model absolute difference is >2e-2: 17.709423065185547
```

Same error for ALBERT v1 models.

## Expected behavior

Max absolute difference between models outputs should be <= 2e-2

## Environment info

Observed on both of the following environments: 

- `transformers` version: 2.5.1
- Platform: Darwin-19.3.0-x86_64-i386-64bit
- Python version: 3.7.6
- PyTorch version (GPU?): 1.4.0 (False)
- Tensorflow version (GPU?): 2.1.0 (False)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

- `transformers` version: 2.5.1
- Platform: Linux-4.15.0-58-generic-x86_64-with-debian-buster-sid
- Python version: 3.7.4
- PyTorch version (GPU?): 1.4.0 (False)
- Tensorflow version (GPU?): 2.1.0 (False)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

**Comment by LysandreJik** [comment]
I could reproduce and I'm currently working on a fix. In the meantime, here's a script that will let you do the export manually, and verify that the conversion worked:

```py
from transformers import AlbertModel, TFAlbertModel, AlbertTokenizer
import os
import numpy as np

os.makedirs(TEMPORARY_LOCATION, exist_ok=True)

tok = AlbertTokenizer.from_pretrained(YOUR_MODEL_HERE)

model = AlbertModel.from_pretrained(YOUR_MODEL_HERE)
model.save_pretrained(TEMPORARY_LOCATION)
tf_model = TFAlbertModel.from_pretrained(TEMPORARY_LOCATION, from_pt=True)

pt = tok.encode_plus("Hi sir how are you doing today", return_tensors="pt")
tf = tok.encode_plus("Hi sir how are you doing today", return_tensors="tf")

pt_ = model(**pt)[0]
tf_ = tf_model(tf)[0]


def difference_between_tensors(tf_tensor, pt_tensor):
    tf_np = np.array(tf_tensor)
    pt_np = np.array(pt_tensor.detach())
    return np.max(np.abs(tf_np - pt_np))

assert difference_between_tensors(tf_, pt_) < 1e-5, "Models are not equivalent"
print("Success!", difference_between_tensors(tf_, pt_))
```

**Comment by singletongue** [comment]
Closing since it seems solved in #4076. Thank you!
